lib/std/pkgconfig.nim

```diff
@@ -0,0 +1,43 @@
+#
+#
+#            Nim's Runtime (and compile-time) Library
+#        (c) Copyright 2024 Nim contributors
+#
+#    See the file "copying.txt", included in this
+#    distribution, for details about the copyright.
+#
+
+## This module implements a compile-time procedure for calling ``pkg-config``.
+##
+## https://www.freedesktop.org/wiki/Software/pkg-config/
+
+from std/envvars import getEnv
+
+proc pkgConfig*(args: string): string {.compiletime.} =
```

Comment:
What about cross platform support?

Reply:
pkg-config doesn't seem to have any flags for cross-compiling.

For cross-compling I would set `$PKG_CONFIG_PATH` to somewhere with pkg-config files for your target, which can be done by calling the compiler with `putenv:PKG_CONFIG_PATH=your-cross-pkg-config-files`.